src/components/Loadable/Loadable.driver.ts

```diff
@@ -0,0 +1,34 @@
+import { ComponentFactory } from 'wix-ui-test-utils/driver-factory';
+import { BaseDriver } from 'wix-ui-test-utils/driver-factory';
+
+export interface LoadableDriver extends BaseDriver {
+  isLoaded: () => boolean;
+  toggleShouldLoadComponents: () => {}
+}
+
+export const loadableDriverFactory = ({
```

Comment:
It looks like a new format and different than the one we have in our drivers, so I wouldn't mix this with all other component drivers.

Reply:
but it's the only way to pass a callback for child driver and selector. Since Loadable could have different childs and not only `Tooltip`. It's a bit different from other components.

Reply:
Not sure I fully understand the use case so we can dive deep into this together, but anyway, since it's just a logical piece, it shouldn't have a driver in this format which we export to consumers. so this can currently stay in the component tests.